test/mocha.opts

```diff
@@ -1,3 +0,0 @@
---require ./node_modules/ts-node/register
```

Comment:
Note this file shouldn't be here as configuring mocha via a `mocha.opts` file is deprecated. Instead a `.mocha.rc.js` file in the root does this.